core/pom.xml

```diff
@@ -196,21 +159,11 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <useSystemClassLoader>true</useSystemClassLoader>
-                    <useManifestOnlyJar>true</useManifestOnlyJar>
+                    <argLine>${additionalRuntimeArgLine}</argLine>
+                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
                     <systemPropertyVariables>
                         <jberet.tmp.dir>${project.build.directory}${file.separator}tmp</jberet.tmp.dir>
                     </systemPropertyVariables>
-                    <!-- ensure we don't inherit a byteman jar from any env settings -->
-                    <environmentVariables>
-                        <BYTEMAN_HOME></BYTEMAN_HOME>
-                    </environmentVariables>
-                    <systemProperties>
-                        <property>
-                            <name>org.jboss.byteman.home</name>
-                            <value></value>
-                        </property>
-                    </systemProperties>
```

Comment:
To be honest I just changed this to mirror the configuration we have in place elsewhere, I didn't give it much thought.
That being said, this configuration I removed essentially forbids anyone to use a custom version of Byteman without changing the pom... If someone sets either BYTEMAN_HOME or the system property `org.jboss.byteman.home`, I'd say they know what they're doing and actually want to override the byteman version, don't they?

I'll put it back if you feel strongly about it (for instance if you know for a fact some people have the `BYTEMAN_HOME` environment variable set system-wide on their machine), but if we do we should rather configure it globally on the whole Hibernate Search project.